src/dali/plugin/input/rest/binance_com.py

```diff
@@ -0,0 +1,940 @@
+# Copyright 2022 macanudo527
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Binance.com REST plugin links:
+# REST API: https://binance-docs.github.io/apidocs/
+# Authentication: https://binance-docs.github.io/apidocs/spot/en/#introduction
+# Endpoint: https://api.binance.com
+
+# CCXT documentation:
+# https://docs.ccxt.com/en/latest/index.html
+
+import datetime as dt
+import json
+import logging
+from datetime import datetime
+from typing import Any, Dict, List, NamedTuple, Optional, Union
+
+import ccxt
+from rp2.logger import create_logger
+from rp2.rp2_decimal import ZERO, RP2Decimal
+
+from dali.abstract_input_plugin import AbstractInputPlugin
+from dali.abstract_transaction import AbstractTransaction
+from dali.dali_configuration import Keyword
+from dali.in_transaction import InTransaction
+from dali.intra_transaction import IntraTransaction
+from dali.out_transaction import OutTransaction
+
+# Native format keywords
+_ACCOUNTPROFITS: str = "accountProfits"
+_ALGO: str = "algo"
+_ALGONAME: str = "algoName"
+_AMOUNT: str = "amount"
+_ASSET: str = "asset"
+_BEGINTIME: str = "beginTime"
+_BUY: str = "buy"  # CCXT
+_COIN: str = "coin"
+_COINNAME: str = "coinName"
+_COST: str = "cost"  # CCXT only variable

```

Comment:
Question on the comments: what is the difference between "CCXT only variable" and "CCXT"?

Reply:
Nothing, just me being lazy and only putting CCXT. I did Find/Replace and made them all "CCXT only variable".

Basically, I just wanted to pre-sort the variables for when we make the abstract CCXT plugin. I probably should have just made a separate list now that I think about it, but will fix it soon.

Reply:
Got it, thanks for the explanation